html.js

```diff
@@ -1,6 +1,5 @@
 "use strict";
 
-var fs = require("fs");
```

Comment:
Huh, if I’d had proper O-Cap discipline, `fs` would be injected below. Oh, well!